TheWarWithin/WarriorFury.lua

```diff
@@ -706,6 +724,11 @@ spec:RegisterStateExpr( "glory_rage", function ()
     return gloryRage
 end )
 
+spec:RegisterHook( "prespend", function( amt, resource, overcap, clean )
```

Comment:
Ok, I think I get it now, this is making Recklessness double the *generation* of Rage.  This probably needs an `and amt < 0` added so that we don't also double the *cost* of Rage for spenders.